### PR TITLE
[Docs] Fix and cleanup issues in Meshery docs

### LINKED ISSUE #3226: [Docs] Fix all bugs in mesheryctl command reference
#### Current State
The command reference page is updated now but still has a lot of bugs like empty examples, empty options, 404 errors on some pages etc. (This is caused due to slight improper handling of YAML and markdown)

#### Desired State
All those errors must be fixed and should be clean if a PR is made on this. This issue is overall fixing of that doc page and not just single change alone!

#### Note
I'll take up this issue and start the work this evening, may take a long time but will try to fix all! ;)

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**
This PR fixes every bug in Meshery docs (mesheryctl and error ref page)

This PR fixes #3226 
Refer: #3149 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by alphaX86** [comment]
Hmm, let's see....
Checklist:
- [x] Fix UI on every doc pages on Options and flags sections
- [x] Fix visible bugs
- [ ] Try to set data via YAML (if possible)
- [ ] Fix mistakes on content

Uh, why me?? :')

**Comment by alphaX86** [comment]
@hexxdump as you reported, the error code bug is fixed

File: docs/pages/reference/mesheryctl-commands.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Meshery CLI commands are categorized by function, which are:
 - `mesheryctl mesh` - Service Mesh Lifecycle & Configuration Management: provisioning and configuration best practices
 - `mesheryctl perf` -  Service Mesh Performance Management: Workload and service mesh performance characterization
 - `mesheryctl pattern` - Service Mesh Pattern Configuration & Management: Service mesh patterns and Open Application Model integration
-- `mesheryctl fitler` - Data Plane Intelligence: Registry and configuration of WebAssembly filters for Envoy
+- `mesheryctl fitler` - Data Plane Intelligence: Registry and configuration of WebAssembly filters for Envoy (Coming soon!)
```
Comment (adithyaakrishna) [inline docs/pages/reference/mesheryctl-commands.md:18]:
```suggestion
- `mesheryctl filter` - Data Plane Intelligence: Registry and configuration of WebAssembly filters for Envoy (Coming soon!)

```

File: docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md [hunk 2]
```diff
@@ -23,11 +23,11 @@ lang: en
 <!-- Basic usage of the command -->
 <pre class="codeblock-pre">
   <div class="codeblock">
-    mesheryctl system status [flags]
+    mesheryctl system status 
   </div>
 </pre>
 
-## Examples
+<!--## Examples
```
Comment (adithyaakrishna) [inline docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md:30]: @alphaX86 I think `mesheryctl system status` requires examples right? 😶 

// @navendu-pottekkat

Comment (adithyaakrishna) [reply docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md:30]: @alphaX86 The title contains, mesheryctl system status | meshery. Can you please remove `| meshery` in this?

Also, this needs to be changed in all the files so title is without `| meshery`
Screenshot from 2021-06-19 20-08-38

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]

File: docs/pages/reference/mesheryctl-commands.md (hunk 1 above)
Comment (alphaX86) [reply docs/pages/reference/mesheryctl-commands.md:18]: I thought I changed that 🙄🤧 alright mate

File: docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md (hunk 2 above)
Comment (alphaX86) [reply docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md:30]: It'd be repeated if I bring examples again

Comment (alphaX86) [reply docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md:30]: I can change the title but regarding that example thing, I really doubt considering UX issues

**Comment by alphaX86** [comment]
Ah c'mon..... DCO not again!

File: docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md (hunk 2 above)
Comment (alphaX86) [reply docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md:30]: Tags removed. We need to decide if this change is good

**Comment by alphaX86** [comment]
Anyone please review what needs to be done 😶

**Comment by adithyaakrishna** [comment]
@alphaX86 Waiting for @navendu-pottekkat 's review regarding the example thingy 😶

**Comment by leecalcote** [comment]
@alphaX86 oof. No small amount of work here! 😄

**Comment by navendu-pottekkat** [comment]
`system status` will only have one example I guess. There is only one way to use the command.

**Comment by alphaX86** [comment]
> `system status` will only have one example I guess. There is only one way to use the command.

That's what, if we include example again after desc. Won't it be repeated?

**Comment by leecalcote** [comment]
@alphaX86 given the size of this PR, perhaps, a demo on this week's dev meeting is in order. Please add the agenda, if you would like to present it.

**Comment by alphaX86** [comment]
Sounds nice @leecalcote I'll add in today's meet agenda right away!

**Pull request merged**